packages/accessibility/test/behaviors/testDefinitions.ts

```diff
@@ -597,7 +597,13 @@ definitions.push({
   regexp: /Triggers '(\w+)' action with '(\w+)' on '([\w-]+)', when has an opened subtree\./g,
   testMethod: (parameters: TestMethod) => {
     const [action, key, elementToPerformAction] = [...parameters.props]
-    const propertyOpenedSubtree = { open: true, items: [{ a: 1 }], siblings: [], hasSubtree: true }
+    const propertyOpenedSubtree = {
+      open: true,
```

Comment:
Shouldn't this open be removed? Also would propose renaming of `propertyOpenedSubtree` to `propertyExpendedSubtree` for consistency.

Reply:
We are not changing the name in the HierarchicalTree too. This was a hack to make this definition work with both behaviors.